### PR TITLE
FIX-#1908: CI fix

### LINKED ISSUE #1908: CI fails on Windows checks
CI fails on Windows tests on the step `Run goanpeca/setup-miniconda@v1.6.0` with the next message:
```
CondaError: Cannot link a source that does not exist. C:\Miniconda\Scripts\conda.exe
Running `conda clean --packages` may resolve your problem.

##[error]The process 'C:\Miniconda\condabin\conda.bat' failed with exit code 1
```
_Note:_ Reopened #1904, since it was closed by mistake

### PR DESCRIPTION
**Pull request by amyskov**
Signed-off-by: Alexander Myskov <alexander.myskov@intel.com>

Additional job was added into CI for Windows tests with `conda clean --packages` command as it was recommended in the error message (see issue for details).

## What do these changes do?

- [x] commit message follows format outlined [here](https://modin.readthedocs.io/en/latest/contributing.html)
- [x] signed commit with `git commit -s` 
- [x] Resolves #1908

### CODE REVIEW

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -1,6 +1,24 @@
...
 on: push
 jobs:
+  prepare_conda_windows: # job is added in order to fix #1908
+    name: prepare_conda_windows
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["windows-latest"]
+        python-version: ["3.6"]
```
Comment (aregm) [inline .github/workflows/push.yml:11]: If we are limiting to only one version of Python, I think we should pick the latest release.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -1,6 +1,24 @@
...
 on: pull_request
 jobs:
+  prepare_conda_windows: # job is added in order to fix #1908
+    name: prepare_conda_windows
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["windows-latest"]
+        python-version: ["3.6"]
```
Comment (aregm) [inline .github/workflows/ci.yml:11]: If we are limiting to only one version of Python, I think we should pick the latest release.

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -1,6 +1,24 @@
 name: ci
 on: pull_request
 jobs:
+  prepare_conda_windows: # job is added in order to fix #1908
+    name: prepare_conda_windows
```
Comment (aregm) [inline .github/workflows/ci.yml:5]: Let's have some more descriptive names, e.g. conda-env.

**Comment by amyskov** [comment]
@aregm it looks like error, that should be fixed by this PR, is not reproduced now, so this PR can be closed, i think.

**Comment by anmyachev** [comment]
@amyskov feel free to reopen this PR if needed

**Pull request closed without merging**